src/ui/components/results/alternativeverticalscomponent.js

```diff
@@ -121,8 +118,11 @@ export default class AlternativeVerticalsComponent extends Component {
     if (typeof referrerPageUrl === 'string') {
       params[StorageKeys.REFERRER_PAGE_URL] = referrerPageUrl;
     }
+    if (sessionsOptIn && sessionsOptIn.setDynamically) {
```

Comment:
Is the purpose of `setDynamically` to save us from this one params assignment? Was this causing a bug?